From 51908b138907d25df02687c514d59373b2a32f18 Mon Sep 17 00:00:00 2001
From: Stefan Wahren <wahrenst@gmx.net>
Date: Sat, 21 Sep 2019 15:55:19 +0200
Subject: [PATCH] ARM: dts: bcm2708-rpi: Define the downstream HDMI
 power at one place

In order to keep downstream definition of the HDMI power domain separate
and independent from upstream changes, add them to bcm2708-rpi.dtsi.

Signed-off-by: Stefan Wahren <wahrenst@gmx.net>
---
 arch/arm/boot/dts/bcm2708-rpi.dtsi | 1 +
 1 file changed, 1 insertion(+)

--- a/arch/arm/boot/dts/bcm2708-rpi.dtsi
+++ b/arch/arm/boot/dts/bcm2708-rpi.dtsi
@@ -102,6 +102,7 @@
 };
 
 &hdmi {
+	power-domains = <&power RPI_POWER_DOMAIN_HDMI>;
 	status = "disabled";
 };
 
